installer/lib/phoenix_new.ex

```diff
@@ -408,7 +409,8 @@ defmodule Mix.Tasks.Phoenix.New do
       prod: [database: "db/#{app}_prod.sqlite"],
       test_begin: "Ecto.Adapters.SQL.begin_test_transaction(#{module}.Repo)",
       test_restart: "Ecto.Adapters.SQL.restart_test_transaction(#{module}.Repo, [])",
-      migration: true}
+      migration: true,
+      sample_binary_id: "11111111-1111-1111-1111-111111111111"}
```

Comment:
No. We should only specify if if binary_id is given and it does not use UUID in its implementation.